packages/cache/package-lock.json

```diff
@@ -1,4210 +1,577 @@
 {
   "name": "@actions/cache",
   "version": "1.0.8",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Thanks @kaylareopelle.  Could you do this update with npm v6?  We're still on Node 12 in this project.